mmdet/datasets/pipelines/transforms.py

```diff
@@ -2002,15 +2004,20 @@ def __init__(self,
                  min_bbox_size=0,
                  bbox_clip_border=True,
                  skip_filter=True,
-                 pad_val=114):
+                 pad_val=114,
+                 prob=0.5):
```

Comment:
```suggestion
                 prob=1.):
```

Reply:
![image](https://user-images.githubusercontent.com/27466624/159616758-763ec608-05c8-4815-801a-657683da02d8.png)

ZwwWayne gave this suggestion that 0.5 might be better.

Reply:
I also think the default value set to 1 has the least impact.

Reply:
Change it to 1 because there cannot be bc breaking